vocabularies/gsq-sites.ttl

```diff
@@ -0,0 +1,139 @@
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix gsqst: <http://linked.data.gov.au/def/gsq-sites/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/gsq-sites> a owl:Ontology .
+
+gsqst:conceptScheme a skos:ConceptScheme ;
+    dct:created "2020-02-05T15:48:27+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2020-02-05T15:48:27+10:00"^^xsd:dateTime ;
+    dct:source "Compiled by the Geological Survey of Queensland" ;
+    skos:definition "The proximate geological features and locations that may be sampled or observed to determine the geological properties of an ultimate feature of interest."@en ;
+    skos:hasTopConcept gsqst:base-station,
+        gsqst:borehole,
+        gsqst:field-site,
+        gsqst:mine,
+        gsqst:project-location,
+        gsqst:seismic-survey-area,
+        gsqst:seismic-survey-line,
+        gsqst:wellbore,
+        gsqst:wellbore-interval ;
+    skos:prefLabel "GSQ Sites"@en .
+
+gsqst:atmospheric-sampling-site a skos:Concept ;
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which an air sample is taken or observation is made."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Atmospheric Sampling Site"@en .
+
+gsqst:fossil-site a skos:Concept ;
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which a fossil sample is taken or an observation made."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Fossil Site"@en .
+
+gsqst:outcrop a skos:Concept ;
+    skos:altLabel "Dig site"@en ;
+    skos:broader gsqst:field-site ;
+    skos:definition "The part of a rock formation that appears at the surface of the ground. Source: Merriam-Webster Dictionary"@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Outcrop"@en .
+
+gsqst:sediment-site a skos:Concept ;
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which a sediment sample is taken or observation is made."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Sediment Site"@en .
+
+gsqst:soil-site a skos:Concept ;
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which a soil sample is taken or observation is made."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Soil Site"@en .
+
+gsqst:stream-site a skos:Concept ;
+    skos:altLabel "Brook"@en,
+        "Canal"@en,
+        "Creek"@en,
+        "River"@en,
+        "Watercourse"@en ;
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which a water sample is taken or observation is made."@en ;

```

Comment:
is it a water sample?  Or a stream sediment geo chem?

Reply:
Change to water site? add "stream sediment site" as an altLabel to sediment site, or a narrower concept?

Reply:
Best to confirm with @CourteneyD or Joe Tang about the best wording here.  I know not final, but best logic we can create as a start is what i would prefer

Reply:
Hmmm - you can take a sediment site from a stream site - I think you could add 'stream sediment site' as an alt label to sediment site to start.

Currently we have a table called site type - which says creek, outcrop, scree etc and then you take a sediment sample or a water sample from that creek.  The site isn't defined by the sample type?

(maybe not helpful)